app/permissions.go

```diff
@@ -101,6 +102,31 @@ func (a *App) ExportPermissions(w io.Writer) error {
 
 	}
 
+	defaultRoleNames := []string{}
+	for _, dr := range model.MakeDefaultRoles() {
+		defaultRoleNames = append(defaultRoleNames, dr.Name)
+	}
+
```

Comment:
I'm seeing the code and it's a bit overkill to generate de default roles to get the names. Actually for the default roles the ID is equal to the Name, so I would change the model/role.go to declare a constant DEFAULT_ROLE_NAMES with the list of default roles, and in the function MakeDefaultRoles I would use the same constant for the map key and for the Name field value.

Reply:
Something like

```go
const DEFAULT_ROLE_NAMES = []string{
    SYSTEM_USER_ROLE_ID,
    SYSTEM_ADMIN_ROLE_ID,
    SYSTEM_POST_ALL_ROLE_ID,
    SYSTEM_POST_ALL_PUBLIC_ROLE_ID,
    SYSTEM_USER_ACCESS_TOKEN_ROLE_ID,
    TEAM_USER_ROLE_ID,
    TEAM_ADMIN_ROLE_ID,
    TEAM_POST_ALL_ROLE_ID,
    TEAM_POST_ALL_PUBLIC_ROLE_ID,
    CHANNEL_USER_ROLE_ID,
    CHANNEL_ADMIN_ROLE_ID,
}
```